e2-studio-star-rx72n-firmware/libs/rx_frame/src/rx_frame.c

```
@@ -614,6 +766,119 @@ rx_err_t rx_frame_decode(const rx_frame_decoder_t* dec,
+ * } else {
+ *     rx_log_error(TAG, "no sync word within scan window");
+ * }
+ * @endcode
+ *
+ * @see rx_frame_decode() Simple decode without stream recovery
+ * @see internal_find_sync_offset() Bounded sync word scanner
+ * @see k_frame_max_scan_bytes Scan window upper bound
+ *
+ * @since Version 1.1.0
+ */
+rx_err_t rx_frame_decode_with_resync(const rx_frame_decoder_t* dec,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_frame_decode_with_resync' of similar type ('const int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)